pvfactors/engine.py

```diff
@@ -206,10 +197,12 @@ def run_timestep(self, idx):
                     - irradiance_vec[:-1] - isotropic_vec
 
                 # Update surfaces with values
-                for idx, surface in geom_dict.items():
-                    surface.update_params({'q0': q0[idx], 'qinc': qinc[idx],
-                                           'isotropic': isotropic_vec[idx],
-                                           'reflection': reflection_vec[idx]})
```

Comment:
there was a non-impactful conflict between the ``idx`` passed to ``run_timestep()`` and the index of this ``for`` loop